### ISSUE TITLE
Community Feedback: How would you name this feature? 

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/Forum#245]
Dear community,

Thus far, it was only possible to add concentration and fraction observed data in PK-Sim using the following nomenclature:

- Add Observed Data
- Add Fraction Data

![image](https://user-images.githubusercontent.com/1041237/48280915-0db8c000-e423-11e8-8deb-a92ad1852583.png)

This is also present in the import ribbon
![image](https://user-images.githubusercontent.com/1041237/48280872-ee219780-e422-11e8-8c42-e64d4786e719.png)

We [recently added](https://github.com/Open-Systems-Pharmacology/PK-Sim/pull/988) in PK-Sim the possibility to import amount data into a project. This will be released in 7.4.

For now the option looks as follow. However we are not quite content with the nomenclature and wanted to gather feedback from the community

![image](https://user-images.githubusercontent.com/1041237/48281237-f75f3400-e423-11e8-96c2-67c7709114d5.png)

![image](https://user-images.githubusercontent.com/1041237/48281261-080faa00-e424-11e8-8e6c-e8fad1938b83.png)

**Note:** The icon color for Amount import will be updated to blue 

We are debating how to name the import feature to be **clear and consistent**.

## Option 1
Add Amount Data

Should we rename `Add Observed Data` to `Add Concentration Data`?

## Option 2
Add Amount Observed Data 

Should we then rename `Add Observed Data` to `Add Concentration Observed Data`?

## Option 3
The User interface could change where we would offer a menu called Observed Data with sub menu (Concentration, Amount, Fraction)

Thanks for your feedback,

Michael

**Comment by StephanSchaller** [comment]
Why don't we just have "Add observed data (for)..." only and decide the rest (i.e. dimension) during the import process when we map the data file entries here:

![image](https://user-images.githubusercontent.com/25477643/48294040-450a8980-e482-11e8-9c77-62e0e14dfeb0.png)

**Comment by StephanSchaller** [comment]
We can also remove the "Add observed data **for**..."... because the only thing it does is preselecting this entry:

![image](https://user-images.githubusercontent.com/25477643/48294142-ccf09380-e482-11e8-9d3c-fcd37a20e044.png)

**Comment by StephanSchaller** [comment]
Otherwise I would go for your option #3 ;-)

**Comment by NinaHanke** [comment]
I totally agree with Stephan. 
I think the "Add observed data **for**... " is not needed. 
And I also like Option 3 best. 
Concentration, amount and fraction is all observed data and having "Add fraction data" as a separate option has led to confusion of new users over here. 
LG, Nina

**Comment by tobiasK2001** [comment]
I also agree with Stephan and would prefere option 3.
Best, Tobias

**Comment by PavelBal** [comment]
Why not use the same UI as in MoBi? I guess option 3 is exactly how it is in MoBi.

I like "Add observed data for..." because it pre-selects the molecular weight of the compound.

**Comment by msevestre** [comment]
@PavelBal MoBi allows for everything and anything to be imported. It is something that we want to somewhat control in PK-Sim probably. Si this is not quite the same I think 

But indeed, maybe we only need one single entry after all. Depending on what is being imported, we would have either fraction, amount etc..

We could then keep the Add Observed Data For... that would preselect the molecule

So in short, we would remove the Add Fraction Data from the UI and only have one option...

@Yuri05 Thoughts? 

Going to investigate to see if this could create some issues in PK-Sim otherwise

**Comment by msevestre** [comment]
Thanks for your feedback everyone. We have simplified the import with only two options available
Add Observed Data
Add Observed Data For

Depending on the selected file, provided that the file as a format that PK-Sim supports, the data will be imported as Amount, Fraction or Concentration.

The option Add Fraction Data has therefore been removed

Corresponding code changes, can be found [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/pull/996)

Michael
